### PR TITLE
[Docs] Replace deprecated Feature Store code

### PR DESCRIPTION
**Pull request by jillnogold**
[ML-5121](https://iguazio.atlassian.net/browse/ML-5121)

[ML-5121]: https://iguazio.atlassian.net/browse/ML-5121?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/feature-store/feature-store-overview.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The training job automatically generates a set of results and versioned artifact
 
 After you validate the feature vector, use the **online** feature service, based on the 
 NoSQL target defined in the feature set, for real-time serving. For serving, you define a serving class derived from 
-`mlrun.serving.V2ModelServer`. In the class `load` method, call the {py:meth}`~mlrun.feature_store.get_online_feature_service` function with the vector name, which returns 
+`mlrun.serving.V2ModelServer`. In the class `load` method, call the {py:meth}`~mlrun.feature_store.FeatureVector.get_online_feature_service` function with the vector name, which returns
```
Comment (alxtkr77) [inline docs/feature-store/feature-store-overview.md:84]: No 'vector name' parameter is required

File: docs/feature-store/end-to-end-demo/03-deploy-serving-model.ipynb [hunk 2]
```diff
@@ -407,8 +407,8 @@
     "import mlrun.feature_store as fstore\n",
     "\n",
     "# Create the online feature service\n",
-    "svc = fstore.get_online_feature_service(\n",
-    "    \"transactions-fraud-short:latest\", impute_policy={\"*\": \"$mean\"}\n",
+    "svc = \"transactions-fraud-short:latest\".get_online_feature_service(\n",
```
Comment (alxtkr77) [inline docs/feature-store/end-to-end-demo/03-deploy-serving-model.ipynb:410]: This line is wrong and will not compile by python

Comment (jillnogold) [reply docs/feature-store/end-to-end-demo/03-deploy-serving-model.ipynb:410]: What should it be?

Comment (alxtkr77) [reply docs/feature-store/end-to-end-demo/03-deploy-serving-model.ipynb:410]: "svc = fstore.get_feature_vector(\"transactions-fraud-short:latest\").get_online_feature_service(impute_policy=...")

**Review by alxtkr77 — APPROVED** [review]

**Pull request merged**